netcontrold/tests/lib/test_util.py

```diff
@@ -262,3 +262,35 @@ def test_rr_cpu_in_numa_positive(self, mock_open):
         out = util.rr_cpu_in_numa()
         expected = [0, 2, 1, 3]
         self.assertEqual(out, expected)
+
+
+class TestUtil_test_thread_creation(TestCase):
+
+    def setUp(self):
+        util.Memoize.forgot = True
+        thread = util.Thread()
+
+    def check_thread_exist(thread):
+        if threading.activeCount() > 0:
+            value1 = 'T'
+        else:
+            value1 = 'F'
+
+        count = util.Thread_status()
+        if count > 0:
+            value2 = 'T'
+        else:
+            value2 = 'F'
+
+        self.assertEqual(value1, value2)
+
+
+class TestUtil_test_searvice_creation(TestCase):
+
+    def setUp(self):
+        util.Memoize.forgot = True
+        service = util.Service()
+
+    def check_pid_exist():
```

Comment:
Like in above thread tests I showed, create few service tests by actually using 'dummy call back' as we did in ncd_ctl and test its existence and expected dummy op etc.